src/components/SavedNews.js

```diff
@@ -1,15 +1,43 @@
-import React, { useState } from 'react';
+import React, { useEffect } from 'react';
 import SavedNewsHeader from './SavedNewsHeader';
 import Footer from './Footer';
 import Search from './Search';
 import Temp from './Temp';
 import FooterMobile from './FooterMobile';
+import MainApi from '../utils/MainApi';
+import { useState } from 'react/cjs/react.development';
 
 function SavedNews(props) {
+  const [cards, setCards] = useState([]);
+  const [numberOfArticles, setNumberOfArticles] = useState(cards.length);
+
+  useEffect(() => {
+    setNumberOfArticles(cards.length);
+  }, [cards.length]);
+
+  useEffect(() => {
+    MainApi.getSavedArticles(props.jwt)
+      .then((res) => {
+        let body = res;
+        setCards(body);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, [props.jwt]);
+
   function handleLogInClick() {
     props.setIsLoggedInFormOpen(true);
   }
 
+  function deleteArticle(cardId) {
+    MainApi.deleteArticle(cardId, props.jwt).then((res) => {
+      const card = document.getElementById(cardId);
+      setNumberOfArticles(numberOfArticles - 1);
+      card.remove();
```

Comment:
you can't delete cards in such a way in React. 

You need to filter the array and remove the card from the array rather than from `DOM`